tests/test_reporter.py

```diff
@@ -14,34 +11,116 @@
 # You should have received a copy of the GNU General Public License along with
 # this program; if not, write to the Free Software Foundation, Inc., 51
 # Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+"""Test cases for reporter module."""
+import StringIO
+import gzip
+import os
 import re
+import shutil
+import tempfile
 import unittest
 
 from contextlib import contextmanager
+import xml.etree.ElementTree as etree
 
 import mock
+import responses
 
 from skt import reporter
 
 from tests import misc
 
 
+SCRIPT_PATH = os.path.dirname(os.path.realpath(__file__))
+
+
+def read_asset(filename):
+    """Read a test asset."""
+    filename = "{}/assets/{}".format(SCRIPT_PATH, filename)
+    with open(filename, 'r') as fileh:
+        return fileh.read()
+
+
+class TestReporterFunctions(unittest.TestCase):
+    """Test cases for the functions in skt.reporter."""
+
+    def setUp(self):
+        """Test fixtures."""
+        self.statefile = "{}/assets/testing_state.cfg".format(SCRIPT_PATH)
+        self.statefile_no_runner = (
+            "{}/assets/testing_state_no_runner.cfg".format(SCRIPT_PATH)
+        )
+
+    def test_load_state_cfg(self):
+        """Test load_state_cfg() with an empty cfg."""
```

Comment:
What does `with an empty cfg` mean here? Is it just a residue of previous implementation?

Reply:
Ah, I meant to remove that.